ngraph/core/src/op/abs.cpp

```diff
@@ -36,8 +36,12 @@ op::Abs::Abs(const Output<Node>& arg)
 
 shared_ptr<Node> op::Abs::clone_with_new_inputs(const OutputVector& new_args) const
 {
-    check_new_args_count(this, new_args);
-    return make_shared<Abs>(new_args.at(0));
+    NGRAPH_OP_SCOPE(Abs_clone_with_new_inputs)
```

Comment:
Align name with evaluate `v0_Abs_evaluate` from `Abs_clone_with_new_inputs` to `v0_Abs_clone_with_new_inputs`

Reply:
Yes I will check all names. It has happened because I used macro to automatically add all macros and the method signature dosn't have `v0` prefix.

Reply:
Fixed